test/Feature/BFSSearcherAndDFSSearcherInterleaved.c

```diff
@@ -0,0 +1,44 @@
+// RUN: %llvmgcc %s -emit-llvm -g -c -o %t.bc
+// RUN: rm -rf %t-bfs.klee-out
+// RUN: rm -rf %t-dfs.klee-out
+// RUN: rm -rf %t-bfs-dfs.klee-out
+// RUN: rm -rf %t-dfs-bfs.klee-out
+// RUN: %klee -output-dir=%t-bfs.klee-out -search=bfs %t.bc >%t-bfs.out
+// RUN: %klee -output-dir=%t-dfs.klee-out -search=dfs %t.bc >%t-dfs.out
+// RUN: %klee -output-dir=%t-bfs-dfs.klee-out -search=bfs -search=dfs -use-batching-search -batch-time=120 -batch-instructions=100000 %t.bc >%t-bfs-dfs.out
```

Comment:
@jbuening Please don't use a big value for `batch-instructions`, rather use `-batch-instructions=1` instead.
This command line argument (`-batch-instructions`) defines after how many instructions the searcher is called to select a new state. Therefore the higher the number the less often the functionality you want to test is called.

Also, please don't use `-batch-time`, disable it. For a testing environment, the computation outcome should not depend on the speed of your processor.